modules/govuk/manifests/apps/whitehall.pp

```diff
@@ -91,6 +91,10 @@
 #   The secret used to encode JWT authentication tokens. This value needs to be
 #   shared with authenticating-proxy which decodes the tokens.
 #
+# [*asset_manager_bearer_token*]
+#   The bearer token to use when communicating with Asset Manager.
+#   Default: undef
+#
```

Comment:
Minor: It seems slightly odd not to put this next to the documentation for `publishing_api_bearer_token`. Similarly in the rest of the file, I would've expected to see the two bearer token definitions next to each other unless there's a good reason not to do that.

Reply:
I've moved this option so that the list of all options is alphabetically ordered as, apart from `sentry_dsn` and `jwt_auth_secret`, this seems to be the convention in the file.